app/javascript/dashboard/components/widgets/conversation/ConversationAdvancedFilter.vue

```diff
@@ -267,12 +310,31 @@ export default {
       }
     },
     appendNewFilter() {
-      this.appliedFilters.push({
-        attribute_key: 'status',
-        filter_operator: 'equal_to',
-        values: '',
+      if (this.isFolderView) {
+        this.setQueryOperatorOnLastQuery();
+      } else {
+        this.appliedFilters.push({
+          attribute_key: 'status',
+          filter_operator: 'equal_to',
+          values: '',
+          query_operator: 'and',
+        });
+      }
+    },
+    setQueryOperatorOnLastQuery() {
+      const lastItemIndex = this.appliedFilters.length - 1;
+      this.appliedFilters[lastItemIndex] = {
+        ...this.appliedFilters[lastItemIndex],
         query_operator: 'and',
-      });
+      };
+      setTimeout(() => {
```

Comment:
Using `setTimeout` is an anti-pattern. Why do you need it here? Try to see if you can rely on `nextTick` if you have to